src/web/routes/application/are-you-pregnant/transform-data.js

```diff
@@ -0,0 +1,16 @@
+const { YES } = require('./constants')
+const { toDateString } = require('../common/formatters')
+
+const transformData = (req) => {
+  if (req.body.areYouPregnant === YES) {
+    const expectedDeliveryDate = toDateString(
+      req.body['expectedDeliveryDate-day'],
+      req.body['expectedDeliveryDate-month'],
+      req.body['expectedDeliveryDate-year']
+    )
+    return { areYouPregnant: req.body.areYouPregnant, expectedDeliveryDate: expectedDeliveryDate }
+  }
+  return { areYouPregnant: req.body.areYouPregnant, expectedDeliveryDate: null }
+}
+
+module.exports.transformData = transformData
```

Comment:
Just as a point of interest, this file could be written like this:
```
const transformData = (req) => {
  const { areYouPregnant } = req.body

  if (areYouPregnant === YES) {
    const expectedDeliveryDate = toDateString(
      req.body['expectedDeliveryDate-day'],
      req.body['expectedDeliveryDate-month'],
      req.body['expectedDeliveryDate-year']
    )
    return {
      areYouPregnant,
      expectedDeliveryDate
    }
  }
  
  return {
    areYouPregnant,
    expectedDeliveryDate: null
  }
}
```
obviously very subjective, but as a JS dev I find that more readable (I know I'm not supposed to make subjective readability comments on PRs!!).

Reply:
your version does look better